src/main/java/org/opentripplanner/framework/logging/Throttle.java

```diff
@@ -35,6 +35,10 @@ public static Throttle ofOneSecond() {
     return new Throttle(1000);
   }
 
+  public static Throttle ofOneMinute() {
+    return new Throttle(1000 * 60);
+  }
+
```

Comment:
I think we should refactor here and have one private constructor with `Duration`, but I also have a PR coming up with a change here, so I can do it. I will get a merge conflict but that if fine.